llvm/test/MC/X86/vinsertps_decode.s

```diff
@@ -0,0 +1,7 @@
+# RUN: llvm-mc -triple x86_64-unknown-unknown %s | FileCheck %s
```

Comment:
Add --no_x86_scrub_mem_shuffle arg to check full line

Reply:
I've manually updated the line for now. I did not generate the test with `update_llc_test_checks.py` because I don't think it works with llvm-mc?